2025-10-21 15:54:42,539 INFO    MainThread:25608 [wandb_setup.py:_flush():81] Current SDK version is 0.22.2
2025-10-21 15:54:42,540 INFO    MainThread:25608 [wandb_setup.py:_flush():81] Configure stats pid to 25608
2025-10-21 15:54:42,540 INFO    MainThread:25608 [wandb_setup.py:_flush():81] Loading settings from C:\Users\HP\.config\wandb\settings
2025-10-21 15:54:42,540 INFO    MainThread:25608 [wandb_setup.py:_flush():81] Loading settings from D:\EEG\egg_encoder\wandb\settings
2025-10-21 15:54:42,540 INFO    MainThread:25608 [wandb_setup.py:_flush():81] Loading settings from environment variables
2025-10-21 15:54:42,540 INFO    MainThread:25608 [wandb_init.py:setup_run_log_directory():705] Logging user logs to D:\EEG\egg_encoder\wandb\run-20251021_155442-7696x17f\logs\debug.log
2025-10-21 15:54:42,540 INFO    MainThread:25608 [wandb_init.py:setup_run_log_directory():706] Logging internal logs to D:\EEG\egg_encoder\wandb\run-20251021_155442-7696x17f\logs\debug-internal.log
2025-10-21 15:54:42,541 INFO    MainThread:25608 [wandb_init.py:init():832] calling init triggers
2025-10-21 15:54:42,541 INFO    MainThread:25608 [wandb_init.py:init():837] wandb.init called with sweep_config: {}
config: {'wandb': {'project': 'EEG_Decoupling', 'entity': None, 'name': 'eeg_triplet_train'}, 'data': {'eeg_path': 'D:\\EEG\\egg_encoder\\data\\EEG_data\\eeg_55_95_std.pth', 'image_vec_path': 'D:\\EEG\\egg_encoder\\data\\image_vectors.npy', 'text_vec_path': 'D:\\EEG\\egg_encoder\\data\\text_vectors.npy', 'splits_path': 'D:\\EEG\\egg_encoder\\data\\EEG_data\\block_splits_by_image_all.pth'}, 'training': {'epochs': 100, 'batch_size': 64, 'learning_rate': 0.0001, 'num_workers': 4, 'device': 'cuda', 'alpha': 0.5, 'temperature': 0.1}, 'model': {'model_name': 'eeg_encoder', 'n_linear_layers': 1, 'channel_merge': 'linear', 'n_heads': 8, 'embedding_dim': 512, 'n_channels': 128, 'n_samples': 440, 'encoder_name': 'braindecode_shallow', 'n_filters_time': 40, 'filter_time_length': 25, 'n_filters_spat': 40, 'pool_time_length': 75, 'pool_time_stride': 15, 'drop_prob': 0.5}, '_wandb': {}}
2025-10-21 15:54:42,541 INFO    MainThread:25608 [wandb_init.py:init():880] starting backend
2025-10-21 15:54:44,806 INFO    MainThread:25608 [wandb_init.py:init():883] sending inform_init request
2025-10-21 15:54:44,825 INFO    MainThread:25608 [wandb_init.py:init():891] backend started and connected
2025-10-21 15:54:44,826 INFO    MainThread:25608 [wandb_init.py:init():961] updated telemetry
2025-10-21 15:54:44,855 INFO    MainThread:25608 [wandb_init.py:init():985] communicating run to backend with 90.0 second timeout
2025-10-21 15:54:47,591 INFO    MainThread:25608 [wandb_init.py:init():1036] starting run threads in backend
2025-10-21 15:54:47,664 INFO    MainThread:25608 [wandb_run.py:_console_start():2509] atexit reg
2025-10-21 15:54:47,664 INFO    MainThread:25608 [wandb_run.py:_redirect():2357] redirect: wrap_raw
2025-10-21 15:54:47,664 INFO    MainThread:25608 [wandb_run.py:_redirect():2426] Wrapping output streams.
2025-10-21 15:54:47,664 INFO    MainThread:25608 [wandb_run.py:_redirect():2449] Redirects installed.
2025-10-21 15:54:47,667 INFO    MainThread:25608 [wandb_init.py:init():1076] run started, returning control to user process
2025-10-21 15:54:52,283 INFO    wandb-AsyncioManager-main:25608 [service_client.py:_forward_responses():80] Reached EOF.
2025-10-21 15:54:52,283 INFO    wandb-AsyncioManager-main:25608 [mailbox.py:close():137] Closing mailbox, abandoning 1 handles.
2025-10-21 15:54:54,188 ERROR   MainThread:25608 [redirect.py:_on_write():664] [all runs] error in stdout callback
Traceback (most recent call last):
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\lib\redirect.py", line 662, in _on_write
    cb(written_data)
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\wandb_run.py", line 2430, in <lambda>
    lambda data: self._console_raw_callback("stdout", data),
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\wandb_run.py", line 399, in wrapper
    return func(self, *args, **kwargs)
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\wandb_run.py", line 457, in wrapper_fn
    return func(self, *args, **kwargs)
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\wandb_run.py", line 1595, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\interface\interface.py", line 802, in publish_output_raw
    self._publish_output_raw(o)
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 37, in _publish_output_raw
    self._publish(rec)
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\interface\interface_sock.py", line 43, in _publish
    self._asyncer.run(lambda: self._client.publish(request))
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\lib\asyncio_manager.py", line 136, in run
    return future.result()
  File "C:\Users\HP\AppData\Local\Programs\Python\Python310\lib\concurrent\futures\_base.py", line 458, in result
    return self.__get_result()
  File "C:\Users\HP\AppData\Local\Programs\Python\Python310\lib\concurrent\futures\_base.py", line 403, in __get_result
    raise self._exception
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\lib\asyncio_manager.py", line 219, in _wrap
    return await fn()
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\lib\service\service_client.py", line 38, in publish
    await self._send_server_request(request)
  File "D:\EEG\egg_encoder\.venv\lib\site-packages\wandb\sdk\lib\service\service_client.py", line 64, in _send_server_request
    await self._writer.drain()
  File "C:\Users\HP\AppData\Local\Programs\Python\Python310\lib\asyncio\streams.py", line 371, in drain
    await self._protocol._drain_helper()
  File "C:\Users\HP\AppData\Local\Programs\Python\Python310\lib\asyncio\streams.py", line 167, in _drain_helper
    raise ConnectionResetError('Connection lost')
ConnectionResetError: Connection lost
